scripts/typescript/afrl-to-new-api/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "afrl-transformation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "npx ts-node src/index.ts",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

Comment:
do you think it would be a good idea to add your name or CRIPT to the authors list?

Reply:
I added my name + CRIPT + MIT. Also set the license to MIT too.